requests/confirmation_email.json

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
Was this meant to be committed?

Reply:
I committed this file to help test the change to utils.py, as I wanted to make sure it worked with both the mobile number and email address, but can remove it if needed

Reply:
I don't see why not. I asked Pete to commit the feedback one.

Reply:
Ah makes sense.